QEfficient/peft/peft_model.py

```diff
@@ -0,0 +1,61 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ----------------------------------------------------------------------------
+
+from peft import PeftModelForCausalLM, PeftType
+
+
+class QEffPeftModelForCausalLM(PeftModelForCausalLM):
```

Comment:
How is this class supposed to be used?

Reply:
If this class is not supposed to be used directly by user, we should put errors

Reply:
This is a patched class to fix issues in onnx export, as it's not taking past_key_values as input in the original peft library. This just fixes that issue. It's used only in the pytorch transform (see pytorch_transform.py). I'll add a docstring to explain this.

Reply:
Let's put errors in `__init__`, `from_pretrained`

Reply:
This is like QEff wrapper for LlamaForCausalLM. I don't think we have to raise errors for everything that users don't have to use. There comes a time when a user with different needs wants to use this class.